### PR TITLE
added content for some components

### PR DESCRIPTION
**Pull request by DivyanshSareen**
added content for alert, avatar, badge, button and card element

### CODE REVIEW

**Review by developer-anand — APPROVED** [review]
Good to go.

**Review by zoyatanweer — APPROVED** [review]
clean code. good to merge

File: avatar.html [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- fontawesome -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css"
+        integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <!-- style(s) -->
+    <link rel="stylesheet" href="./style.css">
+    <title>PeaceUI Docs</title>
```
Comment (sudipkundu999) [inline avatar.html:14]: You may change the page title
to PeaceUI | Avatar
and at other occurrences also

**Review by sudipkundu999 — APPROVED** [review]
Good to merge

File: avatar.html (hunk 1 above)
Comment (DivyanshSareen) [reply avatar.html:14]: Sure, will do it in a later commit 👍

**Pull request merged**